### PR TITLE
Add support for importing config rules

### PR DESCRIPTION
**Pull request by bbrouwer**
Implements #107

### CODE REVIEW

File: src/main/java/com/whiteclarkegroup/liquibaselinter/config/Config.java [hunk 1]
```diff
@@ -29,18 +30,44 @@
...
+        this(
+            ignoreContextPatternString != null ? Pattern.compile(ignoreContextPatternString) : null,
+            ignoreFilesPatternString != null ? Pattern.compile(ignoreFilesPatternString) : null,
+            rules,
+            failFast,
+            enableAfter,
+            imports);
+    }
+
+    public Config(Pattern ignoreContextPattern,
```
Comment (luke-rogers) [inline src/main/java/com/whiteclarkegroup/liquibaselinter/config/Config.java:51]: Would be nice if we could reduce the need for all these constructors here. This one only seems to be needed in some tests so could just pass in the extra `imports` parameter for those. As for the below constructor we could just replace that with a method called `withRules` which rebuilds a new `Config` with the rules passed in. We can work around the `String` type for the patterns by calling `toString` on them, which will return their string representation.

**Review by luke-rogers — COMMENTED** [review]
One small comment but looking really good 👍

File: src/main/java/com/whiteclarkegroup/liquibaselinter/config/Config.java (hunk 1 above)
Comment (bbrouwer) [reply src/main/java/com/whiteclarkegroup/liquibaselinter/config/Config.java:51]: If you prefer the `withRules` approach, I'll switch this to use the builder pattern like `RuleConfig` does? That also allows me to handle the `Pattern` thing differently, and I believe more elegantly. It does make the tests a whole lot more readable using the Builder pattern.

Comment (luke-rogers) [reply src/main/java/com/whiteclarkegroup/liquibaselinter/config/Config.java:51]: Builder pattern would also work nicely - go with that.

**Comment by bbrouwer** [comment]
Is there anything more you are expecting from this pull request?

**Comment by luke-rogers** [comment]
Currently the Codacy quality checks are failing. Once they are resolved we should be good to merge.

**Comment by bbrouwer** [comment]
I fixed that Codacy issue and added a small section of documentation. I believe I am ready with this PR.

**Review by luke-rogers — APPROVED** [review]

**Comment by luke-rogers** [comment]
@bbrouwer the build is failing with `/home/travis/build/whiteclarkegroup/liquibase-linter/src/main/java/com/whiteclarkegroup/liquibaselinter/config/ConfigLoader.java:[56,72] cannot find symbol`
https://travis-ci.org/github/whiteclarkegroup/liquibase-linter

Once that is resolved I will merge.

**Comment by bbrouwer** [comment]
I fixed that build issue.

**Review by luke-rogers — APPROVED** [review]

**Review by davidjgoss — APPROVED** [review]

**Comment by bbrouwer** [comment]
What is needed now to get this merged?

**Pull request merged**